app/main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    _command = command.split()
+    if len(_command) != 3:
+        print("Wrong command!")
+        return
+    mv, source_file, new_file = _command
+    if mv != "mv":
+        print('Command should be "mv"')
+        return
+    *path, file_name = new_file.split("/")
+
+    if len(path) == 0:
+        with (
+            open(source_file, "r") as source,
+            open(file_name, "w") as copy_file
+        ):
+            copy_file.write(source.read())
+        os.remove(source_file)
+        return
+
+    path = "/".join(path)
+    if os.path.exists(path):
+        with (
+            open(source_file, "r") as source,
+            open(path + "/" + file_name, "w") as copy_file
+        ):
+            copy_file.write(source.read())
+    else:
+        new_path = os.path.join(path)
+        os.makedirs(new_path)
+        with (
+            open(source_file, "r") as source,
+            open(new_path + "/" + file_name, "w") as copy_file
+        ):
+            copy_file.write(source.read())
+
+    os.remove(source_file)
```

Comment:
DRY 


It's better to avoid repetition in the file opening and writing logic. 
You can refactor the part where the file is opened and copied into a separate function 
to make the code more readable and maintainable.